File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/StatisticsSample/src/com/hp/workpath/sample/statisticsample/fragment/ResponseInterface.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.statisticsample.fragment;

import com.hp.workpath.api.Result;
import com.hp.workpath.api.statistics.StatisticsJobData;

import java.util.List;

public interface ResponseInterface {
    void success(List<StatisticsJobData> info);
    void failure(String msg, Result result);
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/StatisticsSample/src/com/hp/workpath/sample/statisticsample/fragment/ResponseInterface.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.statisticsample.fragment

import com.hp.workpath.api.Result
import com.hp.workpath.api.statistics.StatisticsJobData

interface ResponseInterface {
    fun success(info: List<StatisticsJobData?>)
    fun failure(msg: String?, result: Result?)
}